src/views/Project/Details/Frameworks/FrameworkDetail/EditFrameworkModal/index.js

```diff
@@ -0,0 +1,263 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Faram, { requiredCondition } from '@togglecorp/faram';
+import produce from 'immer';
+
+import NonFieldErrors from '#rsci/NonFieldErrors';
+import TextInput from '#rsci/TextInput';
+import TextArea from '#rsci/TextArea';
+import LoadingAnimation from '#rscv/LoadingAnimation';
+import PrimaryButton from '#rsca/Button/PrimaryButton';
+import Button from '#rsca/Button';
+import Modal from '#rscv/Modal';
+import ModalBody from '#rscv/Modal/Body';
+import ModalHeader from '#rscv/Modal/Header';
+import {
+    RequestClient,
+    requestMethods,
+} from '#request';
+
+import notify from '#notify';
+import _ts from '#ts';
+
+import FrameworkUsersTable from './FrameworkUsersTable';
+
+import styles from './styles.scss';
+
+const propTypes = {
+    frameworkId: PropTypes.number,
+    frameworkDetails: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    frameworkPatchRequest: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    closeModal: PropTypes.func,
+    isPrivate: PropTypes.bool.isRequired,
+    canEditMemberships: PropTypes.bool.isRequired,
+    // eslint-disable-next-line react/no-unused-prop-types
+    onFrameworkDetailsChange: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+    frameworkDetails: {},
+    frameworkId: undefined,
+    frameworkPatchRequest: {},
+    closeModal: () => {},
```

Comment:
Avoid using noOp function where not required

Reply:
This is true.